source/security/client-side-field-level-encryption-guide.txt

```diff
@@ -272,6 +272,23 @@ Additional Dependencies
          * - x64 Support
            - x64 support is required for CSFLE
 
+   .. tab::
+      :tabid: go
+
+      .. list-table::
+         :header-rows: 1
+
+         * - Dependency Name
+           - Description
+
+         * - Go version 1.10+
+           - Version 1.10 or above is required
+
+         * - ``libmongocrypt`` 1.1.0+
+           - `libmongocrypt <https://github.com/mongodb/libmongocrypt#libmongocrypt>`__ version 1.1.0 or greater is
```

Comment:
Nitpick: would be helpful to stay consistent in notation of future versions https://docs.mongodb.com/meta/style-guide/terminology/general-term-guidelines/use-consistent-time-references/
For Java we use:
"JDK 8 and **later**"
So this should probably be "libmongocrypt version 1.1.0 or **later** is required"